src/nupic/data/record_stream.py

```diff
@@ -313,7 +314,7 @@ def appendRecord(self, record, inputRef=None):
 
 
   @abstractmethod
-  def appendRecords(self, records, inputRef=None, progressCB=None):
+  def appendRecords(self, records, progressCB=None):
```

Comment:
@scottpurdy I removed `inputRef` from this function here and in both `stream_reader.py` and `file_stream_reader.py` because nothing was using it.